aea/helpers/dialogue/base.py

```diff
@@ -222,6 +222,7 @@ def __str__(self):
     def __init__(
         self,
         dialogue_label: DialogueLabel,
+        message_class,
```

Comment:
Missing type hint; should be "Type[Message]".

Reply:
ok I wasn't aware how type hints for a class type is. Done

Reply:
Done